examples/seq2seq/README.md

```diff
@@ -0,0 +1,86 @@
+# Sequence-to-sequnce learning example for machine translation
+
+This is a minimal example of sequence-to-sequence learning. Sequence-to-sequence is a learning model that converts an input sequence into an output sequence. You can regard many tasks in the natural language processing field as this type of task, such as machine translation, dialogue and summarization.
+
+In this simple example script, an input sequence is processed by a stacked LSTM-RNN and it is encoded as a fixed-size vector. The output sequence is also processed by another stacked LSTM-RNN. At decoding time, an output sequence is generated using argmax.
+
+
+## Requriement
+
+This example requires additional libraries.
+
+- [NLTK](http://www.nltk.org/).
+- progressbar2
+
+```
+$ pip install nltk progressbar2
+```
+
+## Dataset format
+
+You need to prepare four files.
+
+1. Source language sentence file
+2. Source language vocabulary file
+3. Targe language sentence file
+4. Target language vocabulary file
+
+In the sentence files, each line represents a sentence. In each line, each sentence needs to be separated into words by space characters.
+Since the number of source and target sentences is the same, note that both files need to have the same number of lines.
+
+In vocabulary files each line represents a word. Words which are not in the vocabulary files are treated as special words `<UNKNOWN>`.
+
+
+## Tarin with WMT dataset
+
+First you need to prepare parallel corpus. Download 10^9 French-English corpus from WMT15 website.
+
+http://www.statmt.org/wmt15/translation-task.html
+
+```
+$ http://www.statmt.org/wmt10/training-giga-fren.tar
```

Comment:
before: `http://www.statmt.org/wmt10/training-giga-fren.tar` 
after: `wget http://www.statmt.org/wmt10/training-giga-fren.tar`

Reply:
Thank you!